src/renderer/store/modules/history.js

```diff
@@ -16,6 +18,10 @@ const getters = {
 
   getHistoryCacheById(state) {
     return state.historyCacheById
+  },
+
+  getSortOrder(state) {
```

Comment:
Getters, mutations and actions are global in the store, please use namespaced names whenever possible (tells other developers where it comes from and avoids name conflicts).
```suggestion
  getHistorySortOrder(state) {
```